.pre-commit-config.yaml

```
@@ -1,24 +1,36 @@
 repos:
```

```suggestion
exclude: ^.pixi$
repos:
```

from my experience, `pre-commit run --all` tries to lint _all_ files and ignores the `.gitignore`. could be wrong, though